router/cmd/main.go

```diff
@@ -23,12 +24,27 @@ import (
 	"github.com/wundergraph/cosmo/router/internal/logging"
 )
 
+var (
+	overrideEnv = flag.String("override-env", "", "env file name to override env variables")
+	memprofile  = flag.String("memprofile", "", "write memory profile to this file")
+	cpuprofile  = flag.String("cpuprofile", "", "write cpu profile to file")
+)
+
 func Main() {
-	var overrideEnv string
-	flag.StringVar(&overrideEnv, "override-env", "", "env file name to override env variables")
 	flag.Parse()
 
-	cfg, err := config.LoadConfig(overrideEnv)
+	if *cpuprofile != "" {
```

Comment:
Could you organize these helpers and cli flags in a separate file in core? I'd like to keep main free of logic.

Reply:
Let's do it in a later PR. I wanted to keep all flags in one place and the logic is minimal. Other logic exists too, so this seemed ok.

Reply:
ok